lib/web/fetch/formdata-parser.js

```diff
@@ -15,6 +15,18 @@ const filenameBuffer = Buffer.from('; filename')
 const dd = Buffer.from('--')
 const ddcrlf = Buffer.from('--\r\n')
 
+/**
+ * @param {string} chars
+ */
+function isAsciiString (chars) {
+  for (let i = 0; i < chars.length; ++i) {
+    if ((chars.charCodeAt(i) & ~0x7F) !== 0) {
```

Comment:
Wonder if -0x80 is the same

Reply:
Same, but I think this one is more readable.